### PR TITLE
Fixed PHPDoc

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fixed PHPDoc
| Type?             | improvement
| Category?         | CO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | N/A
| Possible impacts? | N/A

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/24114)

### CODE REVIEW

**Review by NeOMakinG — APPROVED** [review]

File: classes/PaymentModule.php [hunk 1]
```diff
@@ -198,9 +198,9 @@ public function addCheckboxCarrierRestrictionsForModule(array $shops = [])
      * @param string $payment_method Payment method (eg. 'Credit card')
      * @param string|null $message Message to attach to order
      * @param array $extra_vars
-     * @param null $currency_special
+     * @param int|null $currency_special
      * @param bool $dont_touch_amount
-     * @param bool $secure_key
+     * @param string|bool $secure_key
```
Comment (PierreRambaud) [inline classes/PaymentModule.php:203]: I'm afraid, it should be a bool everytime :)

Comment (Progi1984) [reply classes/PaymentModule.php:203]: But in `classes/Cart.php` l.80 it's a string

Comment (PierreRambaud) [reply classes/PaymentModule.php:203]: In fact you're right, both are accepted -_-

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @Progi1984

**Pull request merged**